packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js

```diff
@@ -106,18 +117,22 @@ function CollapsiblePanelHeader(props) {
 	const { header, content, onSelect, onToggle, expanded, t } = props;
 	const headerColumnClass = `col-${header.length}`;
 	const headerItems = header.map((headerItem, index) => {
-		if (Array.isArray(headerItem)) {
-			const elements = headerItem.map(renderHeaderItem);
-			return (
-				<div key={index} className={classNames(css.group, css[headerColumnClass])}>
-					{elements}
-				</div>
-			);
-		}
+		const elements = Array.isArray(headerItem)
+			? headerItem.map(renderHeaderItem)
+			: renderHeaderItem(headerItem);
+
 		return (
-			<div key={index} className={classNames(css[headerItem.className], css[headerColumnClass])}>
-				{renderHeaderItem(headerItem)}
-			</div>
+			/* eslint-disable jsx-a11y/no-static-element-interactions */
+			<div
+				key={index}
+				className={classNames(css.group, css[headerColumnClass])}
+				onClick={onToggle}
+				onKeyPress={onToggle}
+				role="button"
```

Comment:
Why don't you turn the header div into a button (role=button) instead of all elements in the header ?
Plus, the user doesn't know what the buttons do.

My proposal : 
- the header div  manage the toggle, instead of every element in it
- the header should not be focusable, or even have button semantic, so SR doesn't consider it a button. Instead, to toggle the element, there is the caret, which is a button with a clear text alternative.

What do you think @frassinier ?

Reply:
BTW I see a mistake in a11y. 
Can you add 
- an `aria-expanded` attribute on the toggle button to indicate that it is expanded or not  
- aria-controls="panel-id" to indicates that it controls the targeted panel